.gitignore

```diff
@@ -118,3 +118,9 @@ env/
 
 ### Configuration files ###
 .env
+
+### vagrant files ###
+.vagrant
+
+### log file ###
+gunicorn_daemon.log
```

Comment:
Would be good to delete and ignore `geckodriver.log`

Reply:
Your geckodriver.exe should also be added to .gitignore. It is a large binary you don't manage, and also OS specific. Your README instructions do correctly ask the reader to download their own one.

Since they are already being tracked, you can remove them from the repo with e.g. git rm --cached geckodriver.exe.
Same for gunicorn_daemon.log

Reply:
Thank you both